AUTHORS.md

```diff
@@ -4,6 +4,7 @@ The list of contributors in alphabetical order:
 
 - [Adelina Lintuluoto](https://orcid.org/0000-0002-0726-1452)
 - [Agisilaos Kounelis](https://orcid.org/0000-0001-9312-3189)
+- [Alp Tuna](https://orcid.org/0009-0001-1915-3993)
```

Comment:
You can add yourself in the same commit that introduced functionality. It is not necessary to split this into two commits.